src/main/java/L5task2/FizzBuzzGame.java

```diff
@@ -0,0 +1,18 @@
+package L5task2;
+
+public class FizzBuzzGame {
+    public static void main(String[] args) {
+        for (int i = 1; i <= 100; i++) {
```

Comment:
This should be in separate function. Right now you are not able to create test properly because you cannot call anything. Try to refactor your code: put functionality in separate function, instead of system.out placed in you algorithm return value -> thanks to that you will be able to print it outside your function